src/components/AppRouter/AppRouter.tsx

```diff
@@ -4,26 +4,28 @@ import ErrorPage from "../../pages/ErrorPage.tsx/ErrorPage";
 import FavouritesPage from "../../pages/FavouritesPage/FavouritesPage";
 import MainPage from "../../pages/HomePage/HomePage";
 import MovieInfo from "../../pages/MovieInfo/MovieInfo";
+import { Routes } from "../../constants/Routes";
+import ErrorBoundary from "../ErrorrBoundary/ErrorBoundary";
 
 const AppRouter = () => {
   return (
     <Switch>
-      <Route path="/home">
+      <Route path={Routes.HOME}>
         <MainPage></MainPage>
       </Route>
       <Route exact path="/">
-        <Redirect to="/home"></Redirect>
+        <Redirect to={Routes.HOME}></Redirect>
       </Route>
-      <Route path="/favourites">
+      <Route path={Routes.FAVOURITES}>
         <FavouritesPage></FavouritesPage>
       </Route>
-      <Route path="/error">
+      <Route path={Routes.ERROR}>
         <ErrorPage></ErrorPage>
       </Route>
-      <Route path="/movieInfo/:id">
+      <Route path={Routes.MOVIE_INFO_ID}>
         <MovieInfo></MovieInfo>
       </Route>
-      <Redirect to="/error"></Redirect>
+      <Redirect to={Routes.ERROR}></Redirect>
```

Comment:
Better for this page not use redirect, because then user won't understand what went wrong because the url always will match redirect url.
Place such route at the end of switch, it will be better
<img width="214" alt="image" src="https://user-images.githubusercontent.com/58829953/230723741-577b822a-ae3c-4898-b8b0-1836152635f2.png">